apps/04TowersOfHanoi.js

```diff
@@ -16,24 +16,64 @@ function printStacks() {
     console.log("c: " + stacks.c);
 }
 
-function movePiece() {
+function movePiece(startStack, endStack) {
     // Your code here
+    // pop
+    var piece = stacks[startStack].pop();
+    // push
+    stacks[endStack].push(piece);
 
 }
 
-function isLegal() {
+function isLegal(startStack, endStack) {
     // Your code here
+    var startStackArray = stacks[startStack];
+    var endStackArray = stacks[endStack];
+      //var lastElementOfStart = startStackArray[startStackArray.lenghth - 1];
```

Comment:
I like these variables, feel free to resurrect them (although you have a typo in 
>lenghth